holiholic_database_api/src/main/java/com/holiholic/database/controller/MatrixController.java

```diff
@@ -0,0 +1,18 @@
+package com.holiholic.database.controller;
+
+import com.holiholic.database.DatabaseManager;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+@RestController
+public class MatrixController {
+
```

Comment:
Do you think that changing the name into DistanceTimeController is a better approach? It would provide more clarity.

Reply:
actually I checked and in the database_api we should stick only with the `api` folder. So `controller` and `planner` packages should be removed.